src/app/services/mol.js

```diff
@@ -4,9 +4,13 @@ const config = { url: 'https://api.mol.org/1.x/species/info', query: '?scientifi
 
 async function getSpecies(species) {
   const speciesArray = Array.isArray(species) ? species : [ species ];
-  const promises = speciesArray.map(specie => fetch(`${config.url}${config.query}${specie}`).then(d => d.json()));
+  const promises = speciesArray.map(
+    specie => fetch(`${config.url}${config.query}${specie}`).then(d => d.json())
```

Comment:
`specie` is a word that doesn't exist. The var should be `species`. =)

Reply:
📖